students/YingGuo/session06/test_mailroom4.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+
+"""session06, write test code for mailroom"""
+
+from mailroom4 import report
+from mailroom4 import all_letter
+from mailroom4 import donor_list
+from mailroom4 import letter_print
+from mailroom4 import add_new_donor
+from mailroom4 import letter_all
+from mailroom4 import all_letter
```

Comment:
Minor: You can import multiple names in one `from ... import ...` statement by enumerating all the imported names. E.g.,
```
from mailroom4 import report, all_letter, donor_list, letter_print, add_new_donor, letter_all, all_letter
```
Also, even though we learned that `from a_module import *` is discouraged, it's OK to use that if it's guaranteed not to cause any confusion, like in a test case. So I'd say it's OK to use `from mailroom4 import *` in this case.